cooperator_api/readme/USAGE.rst

```diff
@@ -4,7 +4,7 @@ To give access to the API to a structure, go to
 - click create and select a user, save.
 - communicate the API-KEY to the structure.
 
-It is recommended to create a technical user for the structure belonging to the group "Easy My Coop User".
+It is recommended to create a technical user for the structure belonging to the group "TODO User".
```

Comment:
Obvious TODO.

Reply:
? It's a todo for the users of the modules, not for the developper. Therefore TODO becomes "Cooperator User".